### ISSUE TITLE
Simply `axisLabels` (without dataOrder)

### ISSUE THREAD

**Issue by ax3l** [ComputationalRadiationPhysics/openPMD-standard#129]
Related to #125: the naming of the indexes to axes of matrices, aka `axisLabels`, should simply be ordered from fastest to slowest varying index when accessing its memory contiguously.
### Example Matrix for `axisLabels = "x;y;z"`

that order means, that the _flattened_ matrix has `x` as its _fastest varying index_ and `y` as the _slower varying index_ and `z` as the _slowest varying index_:

**Fortran** (_first_ index varies fastest in contiguous memory; ["column-major"](http://docs.scipy.org/doc/numpy-1.10.0/glossary.html#term-column-major))

``` Fortran90
real(kind=8), dimension(size_x, size_y, size_z) :: A

! efficient write access
do z = 1, size_z
   do y = 1, size_y
      do x = 1, size_x
         a(x,y,z) = 1.0
      end do
   end do
end do
```

**C/C++** (_last_ index varies fastest in contiguous memory; ["row-major"](http://docs.scipy.org/doc/numpy-1.10.0/glossary.html#term-row-major))

``` C++
double A[size_z][size_y][size_x];

// efficient write access
for( size_t z = 0; z < size_z; ++z )
    for( size_t y = 0; y < size_y; ++y )
        for( size_t x = 0; x < size_x; ++x )
            A[z][y][x] = 1.0;
```

**Python numpy.ndarray** (_last_ index varies fastest in contiguous memory; ["row-major"](http://docs.scipy.org/doc/numpy-1.10.0/glossary.html#term-row-major))

``` python
# Python arrays are C-like arrays as long as `order` is not changed
# See:
#   http://docs.scipy.org/doc/numpy-1.10.0/reference/generated/numpy.ndarray.html#numpy.ndarray
#   http://docs.scipy.org/doc/numpy-1.10.0/reference/generated/numpy.ndarray.flatten.html#numpy.ndarray.flatten
#   http://docs.scipy.org/doc/numpy-1.10.0/glossary.html#term-row-major
A = np.ones(shape=(size_z,size_y,size_x), dtype=np.float64)

# one should not loop in such a way in python but if one would,
# here is the efficient index access order accessing all elements
# contiguously:
for z in np.arange(size_z, dtype=np.uint):
    for y in np.arange(size_y, dtype=np.uint):
        for x in np.arange(size_x, dtype=np.uint):
            A[z,y,x] = 1.0
```

(see [this stackoverflow answer](http://stackoverflow.com/questions/4716125/accessing-elements-of-a-matrix-row-wise-versus-column-wise/4716339#4716339); [here](https://en.wikipedia.org/wiki/Locality_of_reference#Matrix_multiplication) is a more complex example for the matrix multiplication in C)
### Example Matrix for `axisLabels = "z;y;x"`

just change the namings or the index variables (but keep the loop-to-matrix-index-order otherwise you will create cache misses).

**Fortran** (_first_ index varies fastest in contiguous memory; ["column-major"](http://docs.scipy.org/doc/numpy-1.10.0/glossary.html#term-column-major))

``` Fortran90
real(kind=8), dimension(size_z, size_y, size_x) :: A

! efficient write access
do x = 1, size_x
   do y = 1, size_y
      do z = 1, size_z
         a(z,y,x) = 1.0
      end do
   end do
end do
```

**C/C++** (_last_ index varies fastest in contiguous memory; ["row-major"](http://docs.scipy.org/doc/numpy-1.10.0/glossary.html#term-row-major))

``` C++
double A[size_x][size_y][size_z];

// efficient write access
for( size_t x = 0; x < size_x; ++x )
    for( size_t y = 0; y < size_y; ++y )
        for( size_t z = 0; z < size_z; ++z )
            A[x][y][z] = 1.0;
```
### New proposed wording in the standard

... _soon_

**Comment by ax3l** [comment]
also affects wording for the ordering of `gridSpacing`, etc

**Comment by ax3l** [comment]
Resolved with #194 

See also: https://github.com/openPMD/openPMD-standard/issues/235#issuecomment-2130075297
